store/helper/helper.go

```diff
@@ -718,3 +720,61 @@ func (h *Helper) GetStoresStat() (*StoresStat, error) {
 	}
 	return &storesStat, nil
 }
+
+// GetPDAddr return the PD Address.
+func (h *Helper) GetPDAddr() ([]string, error) {
+	var pdAddrs []string
+	etcd, ok := h.Store.(tikv.EtcdBackend)
+	if !ok {
+		return nil, errors.New("not implemented")
+	}
+	pdAddrs = etcd.EtcdAddrs()
+	if len(pdAddrs) < 0 {
```

Comment:
`len(pdAddrs) < 0` is always false, it never hapen.

Reply:
Changed it to `len(pdAddrs) == 0`.